src/css/awesone-features.scss

```diff
@@ -0,0 +1,100 @@
+.awesone-features {
+  width: 100%;
+  min-height: 675px;
+  background-image: url(../img/background-awesone.png);
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  display: flex;
+  text-align: center;
+  flex-direction: column;
+  padding-top: 140px;
+
+  @include media-query($tablet) {
+    width: 100%;
+    padding: 40px;
+  }
+
+  @include media-query($small) {
+    padding: 20px;
+  }
+}
+
+.awesone-features__sub-heading {
+  margin-bottom: 15px;
+}
+
+.awesone-features__heading {
+  margin-bottom: 90px;
+
+  @include media-query($tablet) {
+    font-size: 33px;
+  }
+
+  @include media-query($small) {
+    margin: 0 0 45px;
+  }
+}
+
+.awesone-features__column {
+  width: 100%;
+  display: flex;
+  justify-content: space-around;
+  align-items: baseline;
+  Flex-wrap: wrap;
```

Comment:
Flex-wrap: wrap; rewrite for beauty to flex-wrap: wrap;

Reply:
What? Flex-wrap: wrap to Flex-wrap: wrap?